content/docs/command-reference/import-url.md

```diff
@@ -168,9 +168,7 @@ produces a regular stage in `dvc.yaml`.
 To illustrate these examples we will be using the <abbr>project</abbr> explained
 in the [Get Started](/doc/start).
 
-<details>
-
-### Click and expand to set up example
+<details title="Click and expand to set up example">
```

Comment:
OK just noticed we're postponing this. Just a note that per #3391 these titles should no longer start in "Click for" nor end in "example". Thanks